tests/rptest/scale_tests/tiered_storage_stability_test.py

```diff
@@ -142,19 +149,25 @@ def test_restarts(self):
             self.logger.info(f"Rolling restarting nodes")
             self.redpanda.rolling_restart_nodes(self.redpanda.nodes, start_timeout=600, stop_timeout=600)
 
-            node = self.redpanda.nodes[0]
-            node_id = self.redpanda.node_id(node)
-            node_str = f"{node.account.hostname} (node_id: {node_id})"
-            # Stop a node, wait for enough time for movement to occur, then
-            # proceed.
 
             # Hard stop, then restart.
+            node, node_id, node_str = self.get_node(0)
             self.logger.info(f"Hard stopping and restarting node {node_str}")
             self.redpanda.stop_node(node, forced=True)
             self.redpanda.start_node(node, timeout=600)
             wait_until(self.redpanda.healthy, timeout_sec=600, backoff_sec=1)
 
+            # Stop a node, wait for enough time for movement to occur, then
+            # restart.
+            node, node_id, node_str = self.get_node(1)
+            self.logger.info(f"Hard stopping node {node_str}")
+            self.redpanda.stop_node(node, forced=True)
+            time.sleep(60)
```

Comment:
nit: use self.unavailable_timeout

Reply:
> maybe we could add a wait_until so we continue when the move occurs, also we fail early if it didn't occur within 60 seconds, maybe this might reduce potential flakyness

The idea behind the sleep here is to let maybe _some_ movement occur, but not necessarily all replicas - or any known number of replicas - have to move. Rather, it's a simulation of a scenario when a node went down for a minute and then got back.